tests/rptest/redpanda_cloud_tests/omb_validation_test.py

```diff
@@ -357,14 +357,14 @@ def assert_no_rejected():
         # Before the test even starts, assert that there are a "small" number of connections.
         # This number is not zero because various things maintain or make occasional connections
         # to cloud clusters like kminion, so I observe typically ~45 connections on an idle T1 and 450
-        # on an idle T7 cluster. So we just use a SWAG of 100 + 1% of the connection target as the
+        # on an idle T7 cluster. So we just use a SWAG of 100 + 2% of the connection target as the
         # threshold above which we complain.
         # If this fails it is probably because some other test left services running
         # against the cluster.
         count_before = self._connection_count()
-        maximum_allowed = int(tier_limits.max_connection_count * 0.01 + 100)
+        maximum_allowed = int(tier_limits.max_connection_count * 0.02 + 100)
```

Comment:
I wonder if this number will correlate with number of brokers too. But it seems to me that the simpler - the better.

Reply:
It does correlate with the number of brokers, yeah, because things like connectors connect to every broker at least for some of these clients. It's hard to know the exact function so I think it's fine to just to make the multiplier high enough that it passes: as long as it's not more than 10%: the test tries to create 110% of the desired connection count, and the limit is 120%, so sort of less than 10% error is needed in order not to hit the connection throttle. So I think 1% to 2% change is OK in that respect.